build/nuke/ArtifactNames.cs

```diff
@@ -0,0 +1,4 @@
+public static class ArtifactNames
```

Comment:
What about renaming to `Artifacts` to have symmetry to the `Projects` class name?

Reply:
There is a `Artifacts` member under the `Build` class, I didn't look much but initially I when I created it with the name `Artifacts` I had to add the qualifier `global::` to avoid conflict and I wanted to avoid that. I will give it a second look.